Communitter/src/main/java/com/communitter/api/service/UserService.java

```diff
@@ -62,5 +77,69 @@ public User updateUserAbout(Long id, String about) {
         return userRepo.save(existingUser);
     }
 
+    @Transactional
+    public List<UserFollowerDto> getFollowersByFollowee(Long followeeId) {
```

Comment:
I think the name of the method can be changed to getFollowerByUserId since it will be more readable

Reply:
i changed it to getFollowersByFolloweeId. in the repository is it also FolloweeId so it is easy to track.